### DISCUSSION TITLE
https://kdheepak.com/blog/pandoc-lua-filter-for-alerts/

### DISCUSSION THREAD

**Discussion by giscus** [kdheepak/kdheepak.github.io#20] [Comments]
# https://kdheepak.com/blog/pandoc-lua-filter-for-alerts/

https://kdheepak.com/blog/pandoc-lua-filter-for-alerts/

**Comment by aubertc** [comment]
Do you share this filter? I couldn't find it.

**Reply by kdheepak** [reply]
Sure! The code is [here](https://github.com/kdheepak/kdheepak.github.io/blob/8e6c244420fb09e2e4a44986e424ee7ed09e96ca/_quarto/alert.lua).

**Reply by aubertc** [reply]
Thanks a lot!
